opensrp-chw/google-services.json

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
I don't think we commit the Google Services JSON file. This should be ignored in `.gitignore`